### PR TITLE
dev-python/neovim-remote: version bump to 2.1.0

### PR DESCRIPTION
**Pull request by AnAverageHuman**

### CODE REVIEW

File: dev-python/neovim-remote/neovim-remote-2.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+DESCRIPTION="A tool that helps control neovim processes"
+HOMEPAGE="https://github.com/mhinz/neovim-remote"
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="
+	dev-python/neovim-python-client[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (mgorny) [inline dev-python/neovim-remote/neovim-remote-2.1.0.ebuild]: I see `entry_points`, so setuptools are RDEPEND too. In fact, `install_requires` says it clearly.

File: dev-python/neovim-remote/metadata.xml [hunk 2]
```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
-	</pkgmetadata>
+    <maintainer type="person">
```
Comment (mgorny) [inline dev-python/neovim-remote/metadata.xml]: Tab indent, please?

File: dev-python/neovim-remote/neovim-remote-2.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/mhinz/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+DESCRIPTION="A tool that helps control neovim processes"
+HOMEPAGE="https://github.com/mhinz/neovim-remote"
+LICENSE="MIT"
+SLOT="0"
+IUSE="test"
```
Comment (mgorny) [inline dev-python/neovim-remote/neovim-remote-2.1.0.ebuild]: RESTRICT="!test? ( test )"

File: dev-python/neovim-remote/neovim-remote-2.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
+
+DEPEND="${RDEPEND}
+	test? ( dev-python/pytest[${PYTHON_USEDEP}] )
+"
+
+python_test() {
+	py.test -v
```
Comment (mgorny) [inline dev-python/neovim-remote/neovim-remote-2.1.0.ebuild]: pytest -vv || die

Comment (AnAverageHuman) [reply dev-python/neovim-remote/neovim-remote-2.1.0.ebuild]: Added the `die`, but didn't notice a difference between `-v` and `-vv` output.

Comment (mgorny) [reply dev-python/neovim-remote/neovim-remote-2.1.0.ebuild]: I think it mostly affects skipped tests or sth.

Comment (mgorny) [reply dev-python/neovim-remote/neovim-remote-2.1.0.ebuild]: Also the dot is deprecated.

**Pull request closed without merging**